ports/llvm/portfile.cmake

```
@@ -122,62 +147,72 @@ vcpkg_configure_cmake(
         -DLLVM_BUILD_EXAMPLES=OFF
         -DLLVM_INCLUDE_TESTS=OFF
         -DLLVM_BUILD_TESTS=OFF
-        # Disable optional dependencies to libxml2 and zlib
+        # Disable optional dependencies to libxml2 and zlib.
         -DLLVM_ENABLE_LIBXML2=OFF
         -DLLVM_ENABLE_ZLIB=OFF
         # Force TableGen to be built with optimization. This will significantly improve build time.
         -DLLVM_OPTIMIZED_TABLEGEN=ON
-        # LLVM generates CMake error due to Visual Studio version 16.4 is known to miscompile part of LLVM.
-        # LLVM_TEMPORARILY_ALLOW_OLD_TOOLCHAIN=ON disables this error.
-        # See https://developercommunity.visualstudio.com/content/problem/845933/miscompile-boolean-condition-deduced-to-be-always.html
-        -DLLVM_TEMPORARILY_ALLOW_OLD_TOOLCHAIN=ON
-        -DLLVM_ENABLE_PROJECTS=${LLVM_ENABLE_PROJECTS}
-        -DLLVM_TARGETS_TO_BUILD=${LLVM_TARGETS_TO_BUILD}
-        -DPACKAGE_VERSION=${VERSION}
-        -DPYTHON_EXECUTABLE=${PYTHON3}
+        "-DLLVM_ENABLE_PROJECTS=${LLVM_ENABLE_PROJECTS}"
+        "-DLLVM_TARGETS_TO_BUILD=${LLVM_TARGETS_TO_BUILD}"
+        -DPACKAGE_VERSION=${LLVM_VERSION}
         # Limit the maximum number of concurrent link jobs to 1. This should fix low amount of memory issue for link.
         -DLLVM_PARALLEL_LINK_JOBS=1
         # Disable build LLVM-C.dll (Windows only) due to doesn't compile with CMAKE_DEBUG_POSTFIX
         -DLLVM_BUILD_LLVM_C_DYLIB=OFF
+    OPTIONS_DEBUG
         -DCMAKE_DEBUG_POSTFIX=d
 )
 
 vcpkg_install_cmake()
-vcpkg_fixup_cmake_targets(CONFIG_PATH share/${PORT})
+
+vcpkg_fixup_cmake_targets(CONFIG_PATH "lib/cmake/llvm" TARGET_PATH "share/llvm")
+file(INSTALL ${SOURCE_PATH}/llvm/LICENSE.TXT DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+
 if("clang" IN_LIST FEATURES)
-    vcpkg_fixup_cmake_targets(CONFIG_PATH share/clang TARGET_PATH share/clang)
+    vcpkg_fixup_cmake_targets(CONFIG_PATH "lib/cmake/clang" TARGET_PATH "share/clang")
```

```suggestion
    vcpkg_fixup_cmake_targets(CONFIG_PATH "share/clang" TARGET_PATH "share/clang")
```

this is where the cmake files for clang are, for some reason.